engine/flink/management/periodic/src/test/scala/pl/touk/nussknacker/engine/management/periodic/DeploymentManagerStub.scala

```diff
@@ -22,8 +22,8 @@ class DeploymentManagerStub extends BaseDeploymentManager {
       version = None,
       allowedActions = Nil,
       icon = UnknownIcon,
-      tooltip = None,
-      description = None,
+      tooltip = "empty",
```

Comment:
I'd rather like strings like `foo`, `dumb` etc. First time when I saw "empty" in your verifications, I thought that this "empty" is some magic string that we have hardcoded in production code

Reply:
Fixed with "dummy".